### PR TITLE
Add support for views in ddldump

### PR DESCRIPTION
**Pull request by dimitrylukashov**
In the future, I would like to add support for triggers, and procedures as well, but this is a start to get the VIEWs in. Moving to a more a object-oriented pattern and using the information schema to get specific information.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ jobs:
       - checkout
       - run: mdl .
 
+  # replace this with pylint or pyfmt
```
Comment (lra) [inline .circleci/config.yml]: just replace it with black, let black format, move on. It's a public repo, so you can't really use pyfmt.

File: ddldump/main.py [hunk 2]
```diff
@@ -29,153 +29,210 @@
...
 from ddldump.constants import VERSION
 from past.builtins import basestring
 
 
-def get_db_connection(url):
-    """
-    Args:
-        url (str)
+class Database(object):
+    """general class for Database operations"""
```
Comment (lra) [inline ddldump/main.py]: pretty cool

File: .circleci/config.yml (hunk 1 above)
Comment (dimitrylukashov) [reply .circleci/config.yml]: Black uses an image we host.

Comment (dimitrylukashov) [reply .circleci/config.yml]: I'll do this in a different PR.

**Comment by dimitrylukashov** [comment]
@lra can you upload this to pypi?

**Comment by lra** [comment]
Need a new version 1st: https://github.com/percolate/ddldump/pull/16

**Pull request merged**